src/main/java/frc/robot/RobotContainer.java

```diff
@@ -131,27 +131,21 @@ public RobotContainer() {
 
   private void configureAutoBindings() {
     NamedCommands.registerCommand("PrepPlace",
-        Commands.sequence(
-            Commands.deferredProxy(
-                () -> subStateMachine.tryState(RobotState.PREP_CORAL_L3))));
+        Commands.sequence(TRY_PREP_CORAL_L3).asProxy());
 
     NamedCommands.registerCommand("PlaceSequence",
         Commands.sequence(
-            Commands.deferredProxy(
-                () -> subStateMachine.tryState(RobotState.SCORING_CORAL).until(() -> !hasCoralTrigger.getAsBoolean())),
+            TRY_SCORING_CORAL.asProxy().until(() -> !hasCoralTrigger.getAsBoolean()),
             Commands.waitSeconds(1.5),
-            Commands.deferredProxy(
-                () -> subStateMachine.tryState(RobotState.NONE).until(() -> !hasCoralTrigger.getAsBoolean()))));
+            TRY_NONE.asProxy().until(() -> !hasCoralTrigger.getAsBoolean())).asProxy());
```

Comment:
I believe this extra asProxy at the end is not needed. I think it's only need directly touching the TRY_COMMANDS
```suggestion
            TRY_NONE.asProxy().until(() -> !hasCoralTrigger.getAsBoolean())));
```